week7/7.3_homework.ipynb

```diff
@@ -2,76 +2,109 @@
-    "- __(2 pts)__ How soft attention shows itself when included in each of DRQN / A2C (see slides for implementation details)?\n",
-    "- __(1pts)__ Compare different update schemes for DRQN and try to implement the history based\n",
-    "- __(0.5 pts)__ Does it help to add recurrent units when in MDP scenario, e.g fully observable \"CartPole-v0\"?\n",
-    "- __(0.5 pts)__ Does it help to stack more / less  images when using recurrent units? \n",
-    "- __(0.5 pts)__ No one said l2 loss is perfect. Implement Huber or MAE loss for DRQN and/or A2C critic and compare it's performance on the game of your choice (pass proper `loss_function` to `get_elementwise_objective()`) .\n",
-    "- __(1 pts)__ See the very end of this notebook. Some of the games (right side) benefit a lot from additional LSTM memory. But others (left side) do not. That is interesting. Pick up one or several games from the left side and try to figure out why A2C performance decreases when adding LSTM to feadforward architecture?\n",
+    "__Submisson__: To ease mutual pain, please submit\n",
+    "- Some kind of readable report with links to your evaluations, gym uploads, investigation results, etc.\n",
+    "- Explicitly state that you took on a bonus task and where to find it [to make sure it is found and graded].\n",
     "\n",
+    "__Grading__: The main purpose (and source of points) for this notebook is your investigation, not squeezing out average rewards from random environments. \n",
     "\n",
+    "Getting near/above state of the art performance on one particular game will earn you some bonus points, but you can get much more by digging deeper into what makes the algorithms tick and how they compare to one another.\n",
     "\n",
-    "__NB__: This homework will be graded mainly according to the amount of effort put in investigation, not on the final performance you could squeeze out of the particular method on the particular game. However if you feel more excited  about tuning hyperparameters and find the questions above boring you may skip question answering and proceed to the risky bonus part below.  \n",
+    "Okay, now brace yourselves, here comes an assignment!\n",
     "\n",
-    "__Tips__: \n",
-    "- when you crop your image in atari games you should also appopriately reduce filter sizes and amount of filters / size of your Q- / policy network\n",
-    "- don't forget that PG methods need regularizaiton for proper training because of inflated gradient variance when policy becomes deterministic."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "deletable": true,
-    "editable": true
-   },
-   "source": [
-    "# II - Bonus \n",
+    "#### 7.1 rock-paper-sarsa\n",
+    "Go to [the first notebook](https://github.com/yandexdataschool/Practical_RL/blob/master/week7/7.1_rock_paper_scissors.ipynb) and implement basic recurrent SARSA.\n",
+    "\n",
+    "_note: there's a bonus task at the end of this notebook_\n",
```

Comment:
also would be great to have concise description of what to expect there and how it will be rewarded